build-system/tasks/changelog.js

```diff
@@ -1,661 +0,0 @@
-/**
- * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS-IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-'use strict';
-
-/**
- * @fileoverview Creates a gulp task that fetches the titles and files
- * of pull requests using the github API, from `branch` up to the last git tag.
- * Only includes pull requests that have code changes and not only markdown,
- * json, and yaml changes.
- */
-
-const argv = require('minimist')(process.argv.slice(2));
-const assert = require('assert');
-const childProcess = require('child_process');
-const colors = require('ansi-colors');
-const config = require('../test-configs/config');
-const extend = require('util')._extend;
-const git = require('gulp-git');
```

Comment:
This package can be removed from here:

https://github.com/ampproject/amphtml/blob/41a29e78367608958c0740391bd7ccb9d3794243/package.json#L95